helix-core/src/main/java/org/apache/helix/HelixManagerProperty.java

```diff
@@ -80,27 +86,32 @@ public long getHealthReportLatency() {
     return _healthReportLatency;
   }
 
-  public RealmAwareZkClient.RealmAwareZkConnectionConfig getZkConnectionConfig() {
-    return _zkConnectionConfig;
+  public RealmAwareZkClient.RealmAwareZkConnectionConfig getRealmAwareZkConnectionConfig() {
```

Comment:
this can break users as they used to get RealmAwareZkClient earlier, and will need to update code to use appropriate